src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx

```diff
@@ -65,23 +68,41 @@ const AddCourseWithInput: FC<AddCoursesWithInputProps> = ({
 
   return (
     <Box sx={styles.container}>
-      <AppButton component={Link} to={authRoutes.myCourses.newCourse.path}>
+      <AppButton
+        component={Link}
+        sx={isMobile ? styles.addBtn : undefined}
```

Comment:
I have not changed that line, because in such case I have TypeScript Error. sx needs to receive the styles or be undefined. In case we are using sx={isMobile && styles.addBtn} we receive style or false.